sos/report/plugins/mongodb.py

```diff
@@ -64,4 +67,73 @@ def setup(self):
             "/var/opt/rh/rh-mongodb*/log/mongodb/mongod.log"
         ])
 
+
+class UbuntuMongodb(MongoDb, DebianPlugin, UbuntuPlugin):
+
+    packages = (
+        'mongodb-server',
+        'mongodb-server-core',
+        'juju-db',
+    )
+
+    files = (
+        '/var/lib/juju/db',
+        '/var/snap/juju-db/current/db',
+    )
+
+    services = (
+        'juju-db',
+        'mongodb',
+    )
+
+    def setup(self):
+        get_juju_info = False
```

Comment:
Isn't this redundant?

Reply:
First time I've used a walrus operator. Not having it assigned before, I thought we may not have it assigned. Looks like it is all good after I ran the stageone tests locally.

Updated the PR accordingly

Reply:
Yeah, I run a few commands in python before I asked that way, to be sure my question is not too foolish :)